asyncvnc.py

```diff
@@ -453,12 +466,14 @@ class Client:
 
     @classmethod
     async def create(
-            cls,
-            reader: StreamReader,
-            writer: StreamWriter,
-            username: Optional[str] = None,
-            password: Optional[str] = None,
-            host_key: Optional[rsa.RSAPublicKey] = None) -> 'Client':
+        cls,
+        reader: StreamReader,
+        writer: StreamWriter,
+        username: Optional[str] = None,
+        password: Optional[str] = None,
+        host_key: Optional[rsa.RSAPublicKey] = None,
+        force_video_mode: Optional[str] = None,
+    ) -> 'Client':
```

Comment:
btw you can avoid this if we add `from __future__ import annotations`, not sure on your py versioning requirements though.